released/packages/coq-fcsl-pcm/coq-fcsl-pcm.2.1.0/opam

```diff
@@ -21,7 +21,8 @@ This library relies on propositional and functional extentionality axioms."""
 build: [make "-j%{jobs}%"]
 install: [make "install"]
 depends: [
-  "coq" { (>= "8.19" & < "8.21~") | (= "dev") }
+  "coq" { (>= "8.19" & < "8.21~") | 
+  | "rocq-core" {>= "9.0" | = "dev"})
```

Comment:
```suggestion
  "coq" {>= "8.19" & < "9.1~"}
```
The `rocq-core` package without the `coq` shim doesn't work with `coq_makefile`, which is assumed by this package's `Makefile`. You can instead ensure the package is compatible with the `coq` shim for Rocq 9.